syncer/handle_error.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syncer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pingcap/dm/dm/command"
+	"github.com/pingcap/dm/dm/pb"
+	parserpkg "github.com/pingcap/dm/pkg/parser"
+	"github.com/pingcap/dm/pkg/terror"
+
+	"github.com/pingcap/parser"
+	"github.com/pingcap/parser/ast"
+	tmysql "github.com/pingcap/parser/mysql"
+
```

Comment:
```suggestion
```

I found in other code we don't need to split 3-rd package from pingcap and others.